packages/venia-concept/validate-environment.js

```diff
@@ -3,6 +3,21 @@ function validateEnvironment(env) {
     const { str, bool, url } = envalid;
 
     const validation = {
+        IMAGE_CACHE_EXPIRES: str({
```

Comment:
I'd like to increasingly move all of our configuration APIs into environment variables, so that Webpack config doesn't have to be as long.